src/SearchField/index.scss

```diff
@@ -91,6 +91,14 @@
   &.pgn__searchfield--external {
     border: none;
 
+    .pgn__searchfield-form {
+      background-color: unset;
+
+      .pgn__searchfield_wrapper {
+        background-color: $search-form-background-color;
```

Comment:
[curious] Given it appears the `Form.Control` component itself has a white background (see screenshot), I'm wondering if the `.pgn__searchfield-form`'s background color set on line 11 is actually needed? 

Is an alternative solution here to remove the `background-color: $search-form-background-color;` on `.pgn__searchfield-form`?

Reply:
In [PR](https://github.com/openedx/paragon/pull/2597) on line 11 we added bg-color to cover issue with the background of labels and buttons, but with an external buttons it doesn`t work properly. If we just remove this line, the previous issue will occur, so the solution - to add styles separately for the searchfield labels and buttons.